stage_descriptions/auth-04-pl7.md

```diff
@@ -0,0 +1,71 @@
+In this stage, you'll add support for creating a new user using the `ACL SETUSER` command.
+
+### Creating a user
+
+The `ACL SETUSER` command can be used to create a new user in the Redis' ACL system. If the user already exists, flags or rules are used to modify the user's properties.
+
+Example usage:
+
+```bash
+# Using `ACL SETUSER` to create a new user 'eric'
+> ACL SETUSER eric
+OK
+
+# Using `ACL SETUSER` to create a new user 'layla'
+> ACL SETUSER layla
+OK
+
+# Since 'eric' is already existing user and no flags/rules are specified, no changes should be made on the user
+> ACL SETUSER eric
+OK
+
+# List the existing users
+> ACL USERS
+1) "default"
+2) "john"
+```
+
+If the user already exists, the `ACL SETUSER username` command without any options or flags does not change anything for the user.
+
+### Tests
+
+The tester will execute your program like this:
+
+```bash
+$ ./your_program.sh
+```
+
+Then, it will send multiple `ACL SETUSER` commands specifying a username each time. For example, the tester may send the following commands:
+ 
+```bash
+$ redis-cli
+> ACL SETUSER foo
+# Expect: +OK\r\n
+OK
+> ACL SETUSER bar
+# Expect: +OK\r\n
+OK
+> ACL SETUSER foo
+# Expect: +OK\r\n
+OK
+```
+
+The tester will then send an `ACL USERS` command.
+
+```bash
+> ACL USERS
+# Expect ["bar", "default", "foo"]
+1) "bar"
+2) "default"
+3) "foo"
+```
+
+In this case, the tester will validate that the response is the array `["bar", "default", "foo"]`.
+
+### Notes
+
+- The usernames in the array must be sorted in an alphabetical order.
+
+- Even though no user with the username `default` exists, your server must respond as if a user with the username `default` exists.
+
+- You don't need to implement the default user. We'll get to that in the later stages.
```

Comment:
This note is a bit vague and probably misleading since it isn't clear what "implement the default user" means. Let's leave it up to the user how they want to handle this, if I were on this stage I'd implement a default user store with `["default"]` as the initial state. In later stages I'd add other properties as required